Std/Data/BitVec/Lemmas.lean

```diff
@@ -287,12 +287,16 @@ theorem not_def {x : BitVec v} : ~~~x = allOnes v ^^^ x := rfl
 @[simp] theorem getLsb_not {x : BitVec v} : (~~~x).getLsb i = (decide (i < v) && ! x.getLsb i) := by
   by_cases h' : i < v <;> simp_all [not_def]
 
+
 /-! ### shiftLeft -/
 
 @[simp] theorem toNat_shiftLeft {x : BitVec v} :
     BitVec.toNat (x <<< n) = BitVec.toNat x <<< n % 2^v :=
   BitVec.toNat_ofNat _ _
 
+@[simp] theorem toFin_shiftLeft {n : Nat} (x : BitVec v) :
+    BitVec.toFin (x <<< n) = BitVec.toFin (BitVec.ofNat v (x.toNat <<< n)) := rfl
+
```

Comment:
This is great.

Reply:
https://github.com/opencompl/std4/pull/22